catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/controller/NotificationController.java

```diff
@@ -132,8 +143,8 @@ public void handleEvent(Event event) throws IllegalArgumentException {
                     NOTIFICATION_TOPIC_DOWNLOADS_COMETD,
                     propMap);
         } else {
-            LOGGER.debug("Session with ID \"{}\" is not connected to the server. "
-                    + "Ignnoring notification", sessionId);
+            LOGGER.debug("Session with ID [{}] is not connected to the server. "
+                    + "Ignoring notification", sessionId);
```

Comment:
I believe the max column width would be exceeded if this were a single String. 
Although, perhaps the message can be pulled out into a String variable? Especially if it gets reused.